### ISSUE TITLE
Why multiple entries for same sentence, same MAPE_support_scaling.

### ISSUE THREAD

**Issue by dhruvghulati-zz** [uclmr/simpleNumericalFactChecker#11]
When looking at population_claims csv, I see multiple entries for some sentences, e.g. the sentence `<location>Morocco</location> has a population of over <number>33 million</number> and an area of 446,550 km2 -LRB- 172,410 sq mi -RRB- .` has 15 duplicate entries, and the overall file has 1583 entries. Is there a reason there are multiple rows for the same sentence/mape scaling parameter?

**Comment by andreasvlachos** [comment]
This should happen only when sentences are repeated. In the example you mention, they all came from the same file (can see this, it is the last column in the excel sheet). I checked the file the came from is is quite big. It could be due to the HTMLtoText extraction. Also note that they are not exactly the same (assuming we are talking about rows 455 to 470): in some cases the number is 32 instead of 33. Maybe the HTML I downloaded had multiple versions of the statement, some of them from before? I have seen a lot of weird stuff in HTML to text conversion.
